src/org-manager/containers/pending-organisations/pending-organisations.component.html

```diff
@@ -41,8 +39,12 @@ <h2 class="govuk-heading-l">
       </tr>
     </thead>
   </table>
+  <ng-template  #noData>
```

Comment:
Check your spacing - you have two spaces before #noData and a redundant space at the end of the span on the next line

Reply:
I've added a comment to say you've changed it. Resolved.